### PR TITLE
Adding notebook checks to qss

### PR DESCRIPTION
**Pull request by pgoiporia**

### CODE REVIEW

**Comment by pgoiporia** [comment]
Turns out none of these notebooks use "service.run", so we can't dry run anyways. Will include `aqt.ipynb` when we have the configs sorted out. Ready for review @vtomole .

File: examples/neutral_atom_compile.ipynb [hunk 1]
```diff
@@ -38,7 +38,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "provider = qss.superstaq_provider.SuperstaQProvider(api_key=\"YOUR API KEY\")"
```
Comment (paaige) [inline examples/neutral_atom_compile.ipynb:41]: I know that if an API key isn't provided to SuperstaQProvider() it checks if the superstaq_api_key environment variable exists and assigns it to that. But should we add a comment specifying this (e.g. provide your api key to the provider if SUPERSTAQ_API_KEY environment variable is not set). Might be overkill but I'm not sure how obvious this is

**Review by paaige — APPROVED** [review]
Approved (my comment above isn't blocking). lgtm.

File: examples/neutral_atom_compile.ipynb (hunk 1 above)
Comment (pgoiporia) [reply examples/neutral_atom_compile.ipynb:41]: Added strings to each of the notebooks!

**Review by paaige — APPROVED** [review]

**Pull request merged**